### DISCUSSION TITLE
can a dynamic route be a directory that contains a subroute?

### DISCUSSION THREAD

**Discussion by sr229** [vercel/next.js#17349] [Help] [answered]
Given that we have the following directory structure:
```
pages/
    api/ 
       post/
            [id]/
                 index.js
                 create.js
```

`[id]` is our dynamic route, and we want to account for multiple routes that would do different things like `/api/post/:id/create`, is it possible to do it this way?

**Comment by lfades** [comment, marked as answer]
Yes. I see nothing wrong with your code 👍
